nomos-services/data-availability/src/lib.rs

```diff
@@ -109,27 +117,43 @@ where
     }
 }
 
-async fn handle_new_blob<B: DaBackend, A: NetworkAdapter<Blob = B::Blob, Reply = [u8; 32]>>(
+async fn handle_new_blob<
+    B: DaBackend,
+    A: NetworkAdapter<Blob = B::Blob, Attestation = [u8; 32]>,
+>(
     backend: &mut B,
     adapter: &A,
     blob: B::Blob,
 ) -> Result<(), DaError> {
     // we need to handle the reply (verification + signature)
-    backend.add_blob(blob)?;
+    backend.add_blob(blob).await?;
     adapter
         .send_attestation([0u8; 32])
         .await
         .map_err(DaError::Dyn)
```

Comment:
Although this is a question not really related to this PR, the `attestation` here means the KZG commitment of the block?

Reply:
Well in this case is an open question still. But anyway, it is just a proof that the node have received a blob. Then the EZ node will retrieve 2/3 of attestations for generating a certificate that the data was dispersed correctly.

Reply:
Aha. I get it. Thank you.